packages/action-menu/src/action-menu.css

```diff
@@ -17,6 +17,7 @@ governing permissions and limitations under the License.
 .icon {
     width: 18px;
     height: 18px;
+    flex-shrink: 0;
```

Comment:
are these styles/values we should eventually pick up from spectrum-css or dna?

Reply:
It could be that our processing for `::slotted()` selectors does need some extension. This is needed in our local styles because for `sp-action-menu` we are manually applying the default `icon` to the `ActionButton` that is eventually the base class for this element. Styles for `::slotted([slot=icon])` already apply this rule, but in that default slot content doesn't match this selector (in that it's not actually slotted). Should I add a tracking issue to  look into adding an options `defaultSelector` entry to our `slot` rules?

Reply:
Can you just add a code comment in this case then so someone reading the code here understands why the override is needed? If this ends up being more than an occasional one-off pattern it'd be worth adding an issue.

Reply:
Done and done. I also added a note on the sizing while I was there. The fact that is was there lead me initially not to delve too deep on this and don't want others to fall into the same trap.